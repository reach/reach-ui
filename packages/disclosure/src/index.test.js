import React from "react";
import renderer from "react-test-renderer";

import Disclosure from "./";

let snapshot = element => {
  let wrapper = renderer.create(element);
  const tree = wrapper.toJSON();
  expect(tree).toMatchSnapshot();
  return tree;
};

describe("<Disclosure />", () => {
  test("renders without crashing", () => {
    snapshot(<Disclosure>Hello world</Disclosure>);
  });
});
