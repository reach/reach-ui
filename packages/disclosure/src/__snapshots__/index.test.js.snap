// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Disclosure /> buttonLabel is rendered inside a trigger button 1`] = `
<h1>
  Hello world
</h1>
`;

exports[`<Disclosure /> renders without crashing 1`] = `
<h1>
  Hello world
</h1>
`;
